### PR TITLE
Add 1st pass at role

### PR DESCRIPTION
**Pull request by dougszumski**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+#
+---
+- name: Install Infiniband and Gluster packages
+  package:
+    name: "{{ item }}"
+    state: present
+  become: yes
+  with_items:
+    - '@InfiniBand Support'
+    - centos-release-gluster
```
Comment (markgoddard) [inline tasks/main.yml]: Does this not need to be in a separate task from `glusterfs-server`? Also, looks like it requires EPEL (which would be another task).

**Comment by dougszumski** [comment]
Thanks for the review. Fair point about it performing two functions. I think before we start extending this and making it more flexible, it's worth considering options which already exist in the community such as some of the tools in ```https://github.com/gluster```.  Perhaps it was better left as a playbook.

**Comment by markgoddard** [comment]
Good point. There's also Galaxy: https://galaxy.ansible.com/list#/roles?page=1&page_size=10&platforms=centos&autocomplete=gluster.

File: tasks/main.yml (hunk 1 above)
Comment (oneswig) [reply tasks/main.yml]: It looks like centos-release-gluster comes from CentOS Extras, not EPEL.

Comment (markgoddard) [reply tasks/main.yml]: Right you are - epel isn't required.

Comment (dougszumski) [reply tasks/main.yml]: I think you're right @oneswig, however EPEL was already installed in the Centos image I deployed, so I'm not sure if we've used any packages from it or not.

File: tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    fstype: xfs
+    opts: -f -i size=512 -n size=8192 -d su=128k,sw=10
+    dev: "/dev/{{ item }}"
+  become: yes
+  with_items: "{{ gluster_cluster_block_devices }}"
+
+- name: Update fstab and mount bricks
+  mount:
+    src: "/dev/{{ item }}"
+    path: "{{ gluster_cluster_volume_base_path }}/{{ gluster_cluster_volume_name }}/{{ item }}"
```
[the remark this answers is not in the document]
Comment (markgoddard) [reply tasks/main.yml]: Does it need a `/brick` at the end, or is that added by gluster?

[the remark this answers is not in the document]
Comment (dougszumski) [reply tasks/main.yml]: The ```brick``` folder gets created in the midst of line 60.

**Comment by dougszumski** [comment]
What do we want to do with this? Shall I move it to p3 appliances?

**Comment by markgoddard** [comment]
If it's on galaxy we can pull it in via requirements.yml

**Comment by dougszumski** [comment]
Unless there are any objections I'll merge this later today then.

**Comment by oneswig** [comment]
Was looking good to me... 👍

**Pull request merged**